### PR TITLE
Hidden input type

### PR DESCRIPTION
**Pull request by MingChen0919**
when adding workflows that have tools including hidden type input, it throws an error. This PR fixes the issue.

### CODE REVIEW

**Comment by spficklin** [comment]
Hi @MingChen0919 Can you provide a link to an example workflow I can use for testing this PR?

**Comment by spficklin** [comment]
Also, @MingChen0919 It looks like you also have code changes from PR #88 mixed in with this PR. Can you remove that code?

**Comment by MingChen0919** [comment]
Here is an example workflow: https://galaxy.bioinfo.wsu.edu/u/mingchen0919/w/test-hidden-input-type-pr.

File: includes/tripal_galaxy.webform.inc [hunk 1]
```diff
@@ -173,36 +173,28 @@ function tripal_galaxy_build_webform($galaxy_id, $workflow_id) {
...
-              'css_classes' => '',
-              'title_display' => 1,
-              'collapsible' => 1,
-              'collapsed' => 1,
-            ],
-            'required' => '0',
-            'weight' => '1000',
-          ];
-          $webform->current_optional_fieldset = $dcid;
-        }
```
Comment (spficklin) [inline includes/tripal_galaxy.webform.inc:205]: I don't understand why this code was changed for addressing the 'hidden' component?  Is this code from a different fix?

Comment (spficklin) [reply includes/tripal_galaxy.webform.inc:205]: Nevermind, this adjument seems to already be in the code :-/ I'm not sure how it got there, but as long as the workflows are still working and the optional settings are showing up I'm okay with this.

**Review by spficklin — APPROVED** [review]

**Pull request merged**